### PR TITLE
Got D 782 - 852

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Stabs for further Dettenberg manuscripts.

### CODE REVIEW

File: SchleswigGottorf/GotD782.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                    <term key="Prayers"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SD" when="2022-04-06">Created entity with rudimentary
```
Comment (eu-genia) [inline SchleswigGottorf/GotD782.xml:74]: i assume you are moving this ms from new? i do not see the deletion in the PR. you cannot create the same ms in a new directory without removing it from its old place

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: SchleswigGottorf/GotD782.xml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                   <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>
+                              <material key="parchment"/>
+                           </support>
+                            <foliation>The manuscript is not foliated.</foliation>
+                         </supportDesc>
+                     </objectDesc>
+                      </physDesc>
```
Comment (eu-genia) [inline SchleswigGottorf/GotD782.xml:49]: I was also wondering if Dettenberg should be mentioned in provenance

File: SchleswigGottorf/GotD782.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply SchleswigGottorf/GotD782.xml:74]: Indeed. Removed.

File: SchleswigGottorf/GotD782.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply SchleswigGottorf/GotD782.xml:49]: If it is acceptable, could you please merge?

**Review by eu-genia — APPROVED** [review]

**Pull request merged**